mitmproxy/flow_export.py

```diff
@@ -2,6 +2,12 @@
 import netlib.http
 from textwrap import dedent
 
+import sys, re
+if (sys.version_info > (3, 0)):
+    from urllib.parse import urlparse
+else:
+    import urlparse
```

Comment:
@mhils started using `six` for py2/3 compatible imports in recent commits to master.
Maybe you could try to adopt it here as well?